tools/scripts/download_model.py

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python
+import kagglehub
+
+print("path: ", kagglehub.model_download("google/bert/tensorFlow2/answer-equivalence-bem"))  # noqa: T201
```

Comment:
I would probably update this config to exclude all Python files under `tools/scripts/` from the linter as opposed to adding `noqa` and things like that.